src/utils/system.rs

```diff
@@ -110,21 +109,29 @@ pub fn load_dotenv() -> DotenvResult<()> {
         return Ok(());
     }
 
-    match env::var("SENTRY_DOTENV_PATH") {
-        Ok(path) => dotenvy::from_path(path),
-        Err(_) => dotenvy::dotenv().map(|_| ()),
+    let custom_dotenv_paths: &[_] = if let Ok(path) = env::var("SENTRY_DOTENV_PATH") {
+        &[path]
+    } else if let Ok(paths) = env::var("SENTRY_DOTENV_PATHS") {
+        &paths
+            .split(",")
+            .map(|path| path.trim())
+            .filter(|path| !path.is_empty())
+            .map(|path| path.to_string())
+            .collect::<Vec<_>>()
+    } else {
+        // Fallback to default dotenv
+        dotenvy::dotenv()
+            .map_or_else(|e| if e.not_found() { Ok(()) } else { Err(e) }, |_| Ok(()))
```

Comment:
Tbh I think this `map_or_else` is really hard to understand, but it was already that way before. This isn't a blocker IMO.